src/main/java/gicha/service/CharacterService.java

```diff
@@ -135,8 +137,8 @@ public Character update(Long id, Character character) {
         gicha.model.character.Character existingCharacter = characterRepository.findById(id)
                 .orElseThrow(CharacterNotFoundException::new);
         existingCharacter.setName(character.getName());
-        existingCharacter.setBirthplace(Birthplace.birthplaceToBirthplaceDto(character.getBirthplace()));
-        existingCharacter.setElement(Element.elementToElementDto(character.getElement()));
+        existingCharacter.setBirthplace(Birthplace.valueOf(character.getBirthplace().getValue()));
+        existingCharacter.setElement(Element.valueOf(character.getElement().getValue()));
```

Comment:
Nechces uz konecne pridat mapstruct, aby tady nemuselo byt tohle napsane? Dyt se to skoro neda cist ..